index.html

```diff
@@ -459,146 +459,53 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
   </section>
 
   <!-- Profiling mechanism -->
-
   <section id="profiling-mechanism">
-    <h2 id="the-profiling-profile">Profiling Mechanism</h2>
+    <h2>Profiling Mechanism</h2>
```

Comment:
This MR does not contain a new profile mechanism section and it is not obvious 
why the entire profiling mechanism section has been deleted. 

Defining a profiling mechanism is part of the requirements for the profile spec,
so deleting this section is not an option.

Reply:
`"profile": "https://www.w3.org/2021/wot/profile/core"`

That's the mechanism.

In my opinion this is all that is needed, and even this should probably be moved to the Thing Description specification and just referenced from here.